src/js/components/visualization/ChartComponent.jsx

```diff
@@ -114,7 +114,8 @@ export class ChartComponent extends Component {
 
           this._chart.drilldown({
             ...chartData,
-            data: (seriesItem || dataItem)[drillDownFieldName]
```

Comment:
Oh... is this in case `seriesItem` is defined but `seriesItem[drillDownFieldName]` is undefined?